donkeycar/management/kivy_ui.py

```diff
@@ -856,16 +861,24 @@ def on_aug_list(self, obj, aug_list):
         if not self.config:
             return
         self.config.AUGMENTATIONS = self.aug_list
-        self.augmentation = ImageAugmentation(self.config, 'AUGMENTATIONS',
-                                              always_apply=True)
+        self.augmentation = ImageAugmentation(
+            self.config, 'AUGMENTATIONS', always_apply=True)
         self.on_current_record(None, self.current_record)
 
     def on_trans_list(self, obj, trans_list):
         if not self.config:
             return
         self.config.TRANSFORMATIONS = self.trans_list
-        self.transformation = ImageAugmentation(self.config, 'TRANSFORMATIONS',
-                                                always_apply=True)
+        self.transformation = ImageTransformations(
+            self.config, 'TRANSFORMATIONS')
```

Comment:
The way I wrote this you have to pass in the list of transformations, not just the name of the config.  I did this so I could merge the lists to get the complete list for inference;
```
transforms = getattr(cfg, 'TRANSFORMATIONS', []) + getattr(cfg, 'POST_TRANSFORMATIONS', [])
self.transformation = ImageTransformations(self.config, transforms)
```
Even here, since there are two list of tranformations, we have to merge them somehow.  So the above code would use the merged list of transformations.

Reply:
Ok, I see below there is a separate on_post_trans_list().

Reply:
Yeah I found this a bit clunky to pass in the config and some attributes of the config. Actually we don't really need the to have flexibility in the attributes, we can just agree on `TRANSFORMATIONS` and `POST_TRANSFORMATIONS` as we are doing it in many other places, too. And doing the logic in the constructor of ImageTransformations.